### ISSUE TITLE
Support `ipv{n}.address=none` for `ovn` networks

### ISSUE THREAD

**Issue by tomponline** [lxc/lxd#10271]
Support `ipv{n}.address=none` for `ovn` networks to allow completely isolated networks.

This would:

- Disable auto IP (DHCP/RA) for the relevant IP protocol(s).
- Disable DNS forwarding for the relevant IP protocol(s)
- Remove the relevant IP(s) from the router's external switch interface.
- Remove the relevant IP(s) from the router's internal switch interface.
- Disable router uplink connection entirely (if both IPv4 and IPv6 were disabled).

**Comment by bryanyork** [comment]
Yes, would love the ability to create switches between containers that can be used to chain interfaces together for QoS/SQM, etc.
By default I get `Error: No allowed uplink networks in project`

**Comment by tomponline** [comment]
semi-related to https://github.com/lxc/incus/pull/1189

**Comment by tomponline** [comment]
see also https://github.com/lxc/incus/pull/1070
